### PR TITLE
Microverse Test

### PR DESCRIPTION
**Pull request by oluwajuwon8**
- Created html file
- Created css file
- setup linter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,72 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Microverse-Test
-A simple test
+
+>A simple test
+
+![screenshot](./screenshot.png)
```
Comment (arslanbisharat) [inline README.md:7]: - [ ] Please add screenshot of the project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,72 @@
...
+
+## Built With
+
+- HTML 
+- CSS
+- Technologies used
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```
Comment (arslanbisharat) [inline README.md]: - [ ] Please add proper link here.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,72 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (arslanbisharat) [inline README.md]: - [ ] Please delete unnecssaery sections.

File: .github/workflows/linters.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (arslanbisharat) [inline .github/workflows/linters.yml:48]: - [x] Please make sure you don't have any failing linter rule. 


You can find this as [General Requirement ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#general-requirements).

**Review by arslanbisharat — CHANGES_REQUESTED** [review]

**Review by mke2111 — APPROVED** [review]
### Hi @oluwajuwon8, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**